canonical-web-and-design/ubuntu.com
# Remove broken captcha, replaced with google recaptcha

## Done

- Remove broken captcha, replaced with google recaptcha
- This is nearly 100% form assembly code, I just removed all the styles
- The form submits fine, but the errors are a bit odd; however, getting a working form up is more important right now.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/contributors/agreement
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Test that the form submits

## Issue / Card

Fixes #7120
